modular_skyrat/master_files/code/datums/components/tippable.dm

```diff
@@ -0,0 +1,22 @@
+// Modular solution for alternative tipping visuals, for silicon/cyborgs.
+// Originally added by Skyrat-tg PR #10894
+/datum/component/tippable/set_tipped_status(mob/living/tipped_mob, new_status = FALSE)
+	// Defer to TG code if the mob isn't a silicon.
+	if (!(istype(tipped_mob, /mob/living/silicon/robot))
```

Comment:
Missing a `)` at the end of this line.

Reply:
Wait, there's an extra and necessary `(` before `istype`. Remove that instead.

Reply:
I should be much more careful next time considering my experience... sorry!